From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 8 Sep 2010 23:14:18 -0700
Subject: [PATCH] Staging: fix Makefile so broadcom driver will actually build.

commit 6b5e0736634d7b3d5f88d801e5a5e456190c42db upstream.

It would be nice if people tested their patches before sending
them to me...

Cc: Henry Ptasinski <henryp@broadcom.com>
Cc: Brett Rudley <brudley@broadcom.com>
Cc: Nohee Ko <noheek@broadcom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/Makefile b/drivers/staging/Makefile
index 2e82589..88a934e 100644
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -14,7 +14,7 @@ obj-$(CONFIG_W35UND)		+= winbond/
 obj-$(CONFIG_PRISM2_USB)	+= wlan-ng/
 obj-$(CONFIG_ECHO)		+= echo/
 obj-$(CONFIG_OTUS)		+= otus/
-obj-$(CONFIG_BRCM80211))	+= brcm80211/
+obj-$(CONFIG_BRCM80211)		+= brcm80211/
 obj-$(CONFIG_RT2860)		+= rt2860/
 obj-$(CONFIG_RT2870)		+= rt2870/
 obj-$(CONFIG_COMEDI)		+= comedi/
-- 
1.7.1

